examples/purchase-tester/src/main/res/values/strings.xml

```diff
@@ -10,4 +10,6 @@
     <string name="continue_text">Continue</string>
     <string name="loving_cat">😻</string>
     <string name="logs">Logs</string>
+    <string name="google_store_is_unavailable">Google Store is unavailable in this build.</string>
+    <string name="amazon_store_is_unavailable">Amazon Store is unavailable in this build.</string>
```

Comment:
Maybe we can tell folks to download the LAT version or build locally here. 
Ideally we give people a what-to-do when the app doesn't do what they're looking for

(not entirely related but still worth a read: https://wix-ux.com/when-life-gives-you-lemons-write-better-error-messages-46c5223e1a2f)

Reply:
Yeah, I didn't want to make the error message too long, but I think it's worth to add something about next steps. Also, great article!